docs/sphinx/developers/generating-test-images.rst

```diff
@@ -229,6 +229,12 @@ with their default values, is shown below.
       * the number of ROIs containing one shape of the given type to generate
       *
 
+\*
+
```

Comment:
This renders weirdly far below the footnote; can we do better?

![footnote](https://user-images.githubusercontent.com/2630707/34609354-4784f97c-f214-11e7-8162-a9d2748121c7.png)